docs/testing/debug-add-ins-overview.md

```diff
@@ -0,0 +1,71 @@
+
+- [Windows](#debug-on-windows)
+- [Mac](#debug-on-mac)
+- [Linux or other Unix variant](#debug-on-linux)
+
+## Debug on Windows
+
+The following provides general guidance to debugging on Windows. There are special instructions for debugging UI-less custom functions in Excel and event-based add-ins in Outlook. See [Special cases in Windows](#special-cases-in-windows) later in this section.
+
+- If you're using Visual Studio, see [Debug Office Add-ins in Visual Studio](../develop/debug-office-add-ins-in-visual-studio.md).
+- If you're using Visual Studio Code, debug using the [Add-in Debugger Extension for Visual Studio Code](debug-with-vs-extension.md).
+- If you're using any other IDE or don't want to debug inside your IDE, use the developer tools that are associated with the browser runtime that add-ins use on your development computer. See one of the following:
+
+    - [Debug add-ins using developer tools for Internet Explorer](debug-add-ins-using-f12-tools-ie.md)
+    - [Debug add-ins using developer tools for Edge Legacy](debug-add-ins-using-devtools-edge-legacy.md)
+    - [Debug add-ins using developer tools in Microsoft Edge (Chromium-based)](debug-add-ins-using-devtools-edge-chromium.md)
+
+For information about which browser runtime is being used, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
+
+### Special cases in Windows
+
+To debug UI-less custom functions on Windows, see [UI-less custom functions debugging](../excel/custom-functions-debugging.md).
+
+To debug event-based add-ins in Outlook, see [Debug your event-based Outlook add-in](../outlook/debug-autolaunch.md). The process requires Visual Studio Code.
+
+## Debug on Mac
+
+The following provides general guidance to debugging on Mac. There are special instructions for debugging UI-less custom functions in Excel. See [Special cases in Mac](#special-cases-in-mac) later in this section.
+
+- If you're using Visual Studio Code, debug using the [Add-in Debugger Extension for Visual Studio Code](debug-with-vs-extension.md).
+- For any other IDE, use the Safari Web Inspector. Instructions are in [Debug Office Add-ins on a Mac](debug-office-add-ins-on-ipad-and-mac.md).
+
+### Special cases in Mac
+
+To debug UI-less custom functions on Mac, see [UI-less custom functions debugging](../excel/custom-functions-debugging.md).
+
+## Debug on Linux
+
+We don't recommend that you develop Office Add-ins on a Linux (or other Unix variant) computer except in the unusual case where you can be sure that all the add-ins users will be accessing the add-in through Office on the web from a Linux computer. You'll need to [sideload the add-in to Office on the web](sideload-office-add-ins-for-testing.md) to test and debug it. Debugging guidance is in [Debug add-ins in Office on the web](debug-add-ins-in-office-online.md).
```

Comment:
Suggestion to make the opening point stand alone as a sentence. 
```suggestion
We don't recommend that you develop Office Add-ins on a Linux (or other Unix variant) computer. If you need to be sure that all the add-ins users can access the add-in through Office on the web from a Linux computer, you'll need to [sideload the add-in to Office on the web](sideload-office-add-ins-for-testing.md) to test and debug it. Debugging guidance is in [Debug add-ins in Office on the web](debug-add-ins-in-office-online.md).
```

Reply:
This would change the meaning. If their dev machine is Linux, the MUST sideload and debug in Office on the web. There's no other way because there's no desktop Office for Linux. The only situation in which we would recommend developing on Linux is when ALL of the end users of the add-in will themselves be using a Linux machine and, hence, using the add-in on Office on the web.

Reply:
Ah, gotcha. I had misread that earlier. Consider highlighting that point of *all* the users being on Linux.
```suggestion
We don't recommend that you develop Office Add-ins on a Linux (or other Unix variant) computer. The only scenario that warrants developing add-ins on Linux is if all the add-in's users will access the add-in through Office on the web from a Linux computer. You'll need to [sideload the add-in to Office on the web](sideload-office-add-ins-for-testing.md) to test and debug it. Debugging guidance is in [Debug add-ins in Office on the web](debug-add-ins-in-office-online.md).
```

Reply:
My rewrite crossed your last suggestion "in the mail". I rewrote it a different way by putting the recommendation into a note at the end.